superset/views/utils.py

```diff
@@ -490,6 +538,26 @@ def check_slice_perms(_self: Any, slice_id: int) -> None:
         viz_obj.raise_for_access()
 
 
+def check_dashboard_perms(_self: Any, dashboard_id_or_slug: str) -> None:
```

Comment:
best practice is to have a unit test for every function, it would be great if you could add some

Reply:
Yes, i agree. but this function is refactored out from dashboard function. it is tested in 
https://github.com/apache/incubator-superset/blob/448a41a4e7563cafadea1e03feb5980151e8b56d/tests/security_tests.py#L665
I assume the old unit tests didn't break will be good enough.

Reply:
a good practice is to incrementally improve the state of the code, however it will be your call here